.CurrentChangelog.md

```diff
@@ -1,190 +1,62 @@
-  * Added routines to push function instances into import objects.
-  * Removed the exported map getter in `StoreManager`. Used the getter from `ModuleInstance` instead.
-  * Added the module name mapping in `StoreManager`.
-* Refactored the VM class.
-  * Returned the reference to function type instead of copying when getting the function list.
-  * Returned the vector of return value and value type pair when execution.
-* Updated the include path for rust binding due to the API headers refactoring.
+* Used `assumingUnreachable` instead of `__builtin_unreachable` to help the compiler to generate better codes.
+* Updated the order of the members in the proposal enumeration.
+* Refactored the instruction class for reducing the memory usage.
+  * Refactored the `WasmEdge::BlockType` into a struct.
+  * Categorized the members of the instruction class into a union.
 
 Documentations:
 
-* Updated the `wasmedge` commands in the [Run](https://github.com/WasmEdge/WasmEdge/blob/master/docs/run.md) and [SIMD documentation](https://github.com/WasmEdge/WasmEdge/blob/master/docs/simd.md)
-* Updated the examples in the [C API documentation](https://github.com/WasmEdge/WasmEdge/blob/master/docs/c_api.md).
-* Updated the examples in the [host function documentation](https://github.com/WasmEdge/WasmEdge/blob/master/docs/host_function.md).
-* Updated the examples in the [external reference documentation](https://github.com/WasmEdge/WasmEdge/blob/master/docs/externref.md).
-
-Bindings:
-
-* Move rust crate from root path to `bindings/rust`.
+* Added the [build-on-Windows-10](docs/build_on_windows.md) documentation.
+* Added the [Reference Book](https://wasmedge.org/book/en/).
+* Updated the [Release process](ReleaseProcess.md).
 
 Tests:
 
-* Updated the core test suite to the newest WASM spec.
-* Updated and fixed the value comarison in core tests.
-* Added `ErrInfo` unit tests.
-* Added instruction tests for turning on/off the old proposals.
-* Moved and updated the `AST` unit tests into `loader`.
-* Moved and updated the `Interpreter` tests into `Executor` folder.
-* Added the unit tests for new APIs.
-* Applied the WasmEdge C API in the `ExternRef` tests.
+* Handled the tests for the 32-bit platforms.
+* Added the spec tests for the `multi-memories` proposal.
+* Added the test cases for `getaddrinfo` host function.
+* Added the interruptible execution tests.
 
 Misc:
 
-* Enabled GitHub CodeSpaces
-* Added `assuming` for `assert` checking to help compiler to generate better codes.
-
-
-Thank all the contributors that made this release possible!
-
-2021, actly, alabulei1, Alex, Antonio Yang, Ashutosh Sharma, Avinal Kumar, blackanger, Chojan Shang, dm4, eee4017, fossabot, hydai, Jayita Pramanik, Kenvi Zhu, luishsu, LuisHsu, MaazKhan711635, Michael Yuan, MileyFu, Nick Hynes, O3Ol, Peter Chang, robnanarivo, Shen-Ta Hsieh, Shreyas Atre, slidoooor, Sylveon, Timothy McCallum, Vikas S Shetty, vincent, Xin Liu, Yi Huang, yiying, YiYing He, Yona, Yukang, 牟展佑
```

Comment:
Please use `git log --pretty="%an" 0.9.0..master| sort | uniq` to get the contributor list.